kafka-ui-api/src/main/java/com/provectus/kafka/ui/service/KafkaService.java

```diff
@@ -205,12 +205,17 @@ private InternalClusterMetrics collectTopicsMetrics(Map<String, InternalTopic> t
 
   private Map<String, InternalTopic> mergeWithConfigs(
       List<InternalTopic> topics, Map<String, List<InternalTopicConfig>> configs) {
-    return topics.stream().map(
-        t -> t.toBuilder().topicConfigs(configs.get(t.getName())).build()
-    ).collect(Collectors.toMap(
-        InternalTopic::getName,
-        e -> e
-    ));
+    return topics.stream()
+        .map(t -> t.toBuilder().topicConfigs(configs.get(t.getName())).build())
+        .map(t -> t.toBuilder().cleanUpPolicy(t.getTopicConfigs().stream()
+            .filter(config -> config.getName().equals("cleanup.policy"))
+            .findFirst()
+            .orElseGet(() -> InternalTopicConfig.builder().value(null).build())
```

Comment:
Are you returning empty topic if no config value? As for me this is incorrect behaviour

Reply:
Should I throw exception here if cleanup.policy is not found?

Reply:
I'll prefer to send unknown in this case